CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ have notable changes.
 
 Changes since v2.35
 
+### Fixes
+- Fix users roles after deletion. This was introduced in the previous release that optimizes cache updates. (#3243)
```

Comment:
`"Fix issue with users roles after application deletion ..."`? (adding "application" into the first part)